pkgs/development/interpreters/python/cpython/default.nix

```
@@ -299,6 +299,13 @@ in with passthru; stdenv.mkDerivation (finalAttrs: {
   ] ++ optionals (pythonOlder "3.12") [
     # https://github.com/python/cpython/issues/90656
     ./loongarch-support.patch
+  ] ++ optionals (pythonAtLeast "3.11" && pythonOlder "3.12") [
```

```suggestion
  ] ++ optionals (isPy3 && pythonOlder "3.12") [
```

This seems to affect Python 3.10 all the same.